src/components/modal/base/modal-title.scss

```diff
@@ -8,3 +8,7 @@
     font-size: 1rem;
     font-weight: bold;
 }
+
```

Comment:
Can this go in the next-step-button css instead of in the modal?

Reply:
Good idea, done!